Sources/Nimble/Expectation.swift

```diff
@@ -1,5 +1,6 @@
 import Foundation
 
+// Deprecated
```

Comment:
I'd have to check if these attributes still cause build errors. If `@available(*, deprecated, message: "...")` causes a build failure, then it's effectively the same as removing the API.